packages/website/src/docs/checkbox.mdx

```diff
@@ -1,16 +1,18 @@
 ---
 title: Checkbox
 component: Checkbox
+description: Useful for allowing users to select one or more options in a form.
 ---
 
-```js live=true
+```js live
```

Comment:
should there be some examples, showing how it looks when disabled, etc?

Reply:
Yeah but we don't have styles for that yet... 🙈